release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -2813,6 +2813,52 @@ This section will continue to be updated over time to provide notes on enhanceme
 ====
 
+// 4.19.2
+[id="ocp-4-19-2_{context}"]
+=== RHSA-2025:9750 - {product-title} {product-version}.2 image release, bug fix, and security update advisory
+
+Issued: 01 July 2025
+
+{product-title} release {product-version}.2, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:9750[RHSA-2025:9750] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2025:9751[RHSA-2025:9751] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.19.2 --pullspecs
+----
+
+[id="ocp-4-19-2-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the installation program checked only the first compute machine pool entry in the install configuration to determine whether to disable the `MachineConfigOperator` (MCO) boot image management. If multiple compute pools were specified (Amazon Web Service (AWS) edge nodes are the only supported scenario), but another compute machine pool had a custom Amazon Machine Image (AMI), the installation program would not disable the MCO boot image management and the custom AMI would be overwritten by the MCO. With this release, the installation program checks all compute machine pool entries and if a custom image is found, the MCO boot image management is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-58060[OCPBUGS-58060])
+
+* Previously, if a user specified a custom boot image for Amazon Web Services (AWS) or the Google Cloud Platform (GCP), the Machine Config Operator (MCO) would overwrite it with the default manage image during installation. With this release, a manifest generation was added for MCO configuration which disables the default managed image during installation if a custom image is specified. (link:https://issues.redhat.com/browse/OCPBUGS-57796[OCPBUGS-57796])
+
+* Previously, a validation issue within `oc-mirror` causes the command to reject the `file://.` reference. Users attempting to use `file://.` for a content path received an error message stating `content filepath is tainted`. With this release, `oc-mirror` properly validates the '.' directory reference. (link:https://issues.redhat.com/browse/OCPBUGS-57786[OCPBUGS-57786])
+
+* The `oc-mirror v2` was not using the correct filtered catalog during its operations leading to errors such as including more operators than specified in the configuration, and trying to connect to the catalog registry during disk-to-mirror workflows even in air-gapped environments. With this release, the correct filtered catalog is used. (link:https://issues.redhat.com/browse/OCPBUGS-57784[OCPBUGS-57784])
```

Comment:
Currently, it _could_ be interpreted that the operations are leading to errors, not the incorrect catalog. 
```suggestion
* The `oc-mirror v2` was not using the correct filtered catalog during its operations, which led to errors such as including more operators than specified in the configuration, and trying to connect to the catalog registry during disk-to-mirror workflows even in air-gapped environments. With this release, the correct filtered catalog is used. (link:https://issues.redhat.com/browse/OCPBUGS-57784[OCPBUGS-57784])
```